### ISSUE TITLE
Implement get metadata for a list of callables in REST API

### ISSUE THREAD

**Issue by zikanl** [fasten-project/fasten#153]
A method to get metadata in REST API is not currently implemented. This method should get relevant metadata from the callables table in FASTEN DB. The metadata requested could be either vulnerability-related, quality-related or both.

**Comment by MihhailSokolov** [comment]
@zikanl how would you want to identify the callables whose metadata you want to obtains? Would the list of callable IDs in the body of the request suffice? Also, do you want to obtain the whole metadata from the callable or only certain attributes such as vulnerabilities and/or quality metrics?

**Comment by zikanl** [comment]
@MihhailSokolov  It would be ideal if we could support a choice of attributes for the metadata. This would allow us to specify whether we want e.g. vulnerabilities only, quality metrics only, or whole metadata.

W.r.t. callables - we need a way to uniquely identify a callable. If callable IDs are enough, then we could follow that way

**Comment by MihhailSokolov** [comment]
Implemented in 56e7831
The POST endpoint `/metadata/callables` takes a list of FASTEN URIs for which you want to retrieve metadata in the body of the request and has optional parameters `attributes: List<String>` where you can specify the exact names of top-level metadata attributes you need and `allAttributes: boolean` if you want to retrieve all metadata attributes.
Later I will add a `namespace` to the body request so that callables can be identified properly (uri + module namespace)

**Comment by MihhailSokolov** [comment]
@zikanl We took a bit different approach and instead of somehow passing package name and version as the parameters we decided to restrict the endpoint to allow only the full FASTEN URIs (`fasten://{forge}!{package_name}${version}/{partial_fasten_uri}`) in the request body. You can use `core.utils.FastenUriUtils` to convert partial URI into full URI and vice versa. The attribute parameters remain the same.
Example:
```
curl -X POST -d '["fasten://mvn!junit:junit$4.13/junit.framework/TestResult.startTest(Test)%2Fjava.lang%2FVoidType"]' -H 'Content-Type: application/json'  http://api.fasten-project.eu/api/metadata/callables?allAttributes=true

{
    "fasten://mvn!junit:junit$4.13/junit.framework/TestResult.startTest(Test)%2Fjava.lang%2FVoidType": {
        "access":"public",
        "defined":true
    }
}
```
If you agree with everything, then I would appreciate if you could close the issue :)
